packages/core/docs/rendering-popup.MD

```diff
@@ -0,0 +1,126 @@
+### This script works only with partial popup
+
+```html
+<script>
+  const popupId = 1; // id from DB
+
+  (function (window, doc) {
+    const popupUrl = `${apiURL}/${popupId}`; // API url to get popup data;
+
+    function _appendScript(data) {
+      const { doc, node, onLoad, onError } = data;
+      const script = doc.createElement("script");
+
+      for (let i = 0; i < node.attributes.length; i++) {
+        const { name, value } = node.attributes[i];
+        script[name] = value;
+      }
+
+      // if the script don't have a src, it means it's inline script
+      if (!script.src) {
+        script.innerHTML = node.innerHTML;
+
+        // the inline script is already loaded
+        onLoad();
+      } else {
+        // attach the onload and onerror event to the script
+        script.onload = onLoad;
+        script.onerror = onError;
+      }
+
+      doc.body.appendChild(script);
+    }
+
+    function _appendStyle(data) {
+      const { doc, node, onLoad, onError } = data;
+      // if the node is a meta tag or a dns-prefetch tag, resolve the promise
+      if (node instanceof HTMLMetaElement || node.rel === "dns-prefetch") {
+        onLoad();
+      } else {
+        // attach the onload and onerror event to the node
+        node.onload = () => onLoad();
+        node.onerror = () => onError();
+      }
+      doc.head.appendChild(node);
+    }
+
+    function _overrideScripts(data) {
+      const { doc, node } = data;
+      const docScript = document.createElement("script");
+      docScript.innerHTML = node.innerHTML;
+
+      for (let i = 0; i < node.attributes.length; i++) {
+        const { name, value } = node.attributes[i];
+        script[name] = value;
+      }
+
+      node.replaceWith(docScript);
+    }
+
+    (async () => {
+      try {
+        // API request returns object with html, styles and scripts
+        const data = await fetch(popupUrl, { method: "GET" });
+        const { html: _html, styles: _styles, scripts: _scripts } = await data.json();
```

Comment:
i think you can combine this in one line

`const { html: _html, styles: _styles, scripts: _scripts } = await fetch(popupUrl, { method: "GET" }).json()`